06.bowling_object/lib/game.rb

```diff
@@ -5,8 +5,7 @@
 
 class Game
   def initialize(marks_text)
-    shots_by_frame = parse_marks(marks_text)
-    @frames = shots_by_frame.map.with_index { |shots, idx| Frame.new(idx, shots) }
+    @frames = parse_marks(marks_text)
```

Comment:
```suggestion
    @frames = build_frames(marks_text)
```

parse_marksでも大きな問題はないですが、こんな感じにした方が「frameの配列を作ったからframesに入れるんだな」ということが明確になっていいかも